File: 7-kyu/all-star-code-challenge-#31/javascript/solution.js
const helpJesse = array => {
  let ret = [];
  for (const inst of array) {
    let s = `Pour ${inst.amount} mL of ${inst.solution} into a ${inst.instrument}`;
    if (inst.note) {
      s += ` (${inst.note})`;
    }
    ret.push(s);
  }
  return ret;
}

File: 7-kyu/all-star-code-challenge-#31/python/solution.py
def translate_step(instruction):
    try:
        suffix = f' ({instruction.note})'
    except AttributeError:
        suffix = ''
    return f'Pour {instruction.amount} mL of {instruction.solution} into a {instruction.instrument}{suffix}'

def help_jesse(arr):
    return [translate_step(step) for step in arr]
